components/dashboard/TopBar.js

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable styled-components-a11y/click-events-have-key-events */
```

Comment:
Not a fan of hiding ESLint errors for an entire file; we tend to forget about them. I prefer:
- Hiding just for the concerned line(s)
- Disabling for an entire section of the code
- Or disabling the rule globally if it doesn't make sense for us

Not a blocker for me, but we should at least add a comment to explain why we're doing this so that the next developer working on this file we know why we did it.

Reply:
Agree. This file is actually just a copy of `/components/admin-panel/TopBar.js` (but importing other constants) - a bit of a temporary tech debt to support both the /dashboard and existing /admin-panel